changelog.d/20331_databend_config_ndjson.enhancement.md

```diff
@@ -0,0 +1,3 @@
+Added new config field `missing_field_as` as file format option for NDJSON encoding, default value is `NULL`.
```

Comment:
```suggestion
Added new config field `missing_field_as` to the `databend` sink as file format option for NDJSON encoding, default value is `NULL`.
```

Reply:
Just curious, what is the current behavior? Is this changing the default?

Reply:
Yes, the default is ERROR, which is too strict.